packages/sanity/src/core/studio/StudioTelemetryProvider.tsx

```diff
@@ -97,5 +97,5 @@ export function StudioTelemetryProvider(props: {children: ReactNode; config: Con
 }
 
 function getWellKnownName(name: string) {
-  return WELL_KNOWN_NAMES.includes(name.toLowerCase()) ? name : undefined
+  return WELL_KNOWN_NAMES.find((wellKnownName) => name.toLowerCase().includes(wellKnownName))
```

Comment:
note that this will return the well known name, and not the actual workspace name/dataset name, so e.g. `foo-Staging` will be logged as `staging`